server/routes/connections.js

```diff
@@ -2,11 +2,12 @@ import _ from 'lodash';
 import { Router } from 'express';
 
 import multipartRequest from '../lib/multipartRequest';
+import { CONNECTIONS_LIST_LIMIT } from "../constants";
```

Comment:
Should this constant be added to the `server/constants.js` file? Now you have it only in the `client/constants.js`

Reply:
good spot - addressed in [340edf0](https://github.com/auth0-extensions/auth0-delegated-administration-extension/pull/292/commits/340edf0170d77539afd0eaea7af2a5ded4e02635)